### PR TITLE
Create calibre-web-nginx.conf

### PR DESCRIPTION
**Pull request by deldesir**
Allows access to /library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>

### CODE REVIEW

**Comment by deldesir** [comment]
I symlink /library/calibre-web with /library/www/html/calibre-web

**Comment by holta** [comment]
Related:

- PR #51

**Comment by holta** [comment]
Should this PR temporarily eliminate its 3 Ansible vars, hard-coding them to {/books, /libros, /libros} during testing of... ?

- PR #51

**Comment by holta** [comment]
> Yes. We don't need them for now though support to handle these hardcore variables in all Calibre-Web routes has been added in PR #33

Are you going ahead hard-coding them to {`/books`, `/libros`, `/libros`} in this PR, so people can test this?

File: scripts/calibre-web-nginx.conf [hunk 1]
```diff
@@ -1,33 +1,3 @@
...
-# http://box/livres
-location {{ calibreweb_url3 }}/ {
-    proxy_set_header        Host            $http_host;
-    proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
-    proxy_set_header        X-Scheme        $scheme;
-    proxy_set_header        X-Script-Name   "{{ calibreweb_url3 }}";
-    proxy_pass http://127.0.0.1:8083;
-}
-
-# http://box/library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>
```
Comment (holta) [inline scripts/calibre-web-nginx.conf]: Interesting.
i.e. removing {`/books`, `/libros`, `/livres`} for now, if only for testing.

(Preserve comment lines ideally!?)

```
# RELATED: https://github.com/janeczku/calibre-web/wiki/Setup-Reverse-Proxy

# http://box/library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>
```

Comment (deldesir) [reply scripts/calibre-web-nginx.conf]: I removed them. We don't need them for testing.

Comment (holta) [reply scripts/calibre-web-nginx.conf]: Understood:

Please restore both in-line comments to help @EMG70 and others what might be changing!

Comment (deldesir) [reply scripts/calibre-web-nginx.conf]: Done

**Pull request merged**